sig-network/annual-report-2023.md

```diff
@@ -4,6 +4,27 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
+  - Governance and leadership changes
+
+    - Casey Davenport (**[@caseydavenport](https://github.com/caseydavenport)**)
+      and Dan Williams (**[@dcbw](https://github.com/dcbw)**) stepped
+      down as Chairs, Shane Utt (**[@shaneutt](https://github.com/shaneutt)**) and
+      Mike Zappa (**[@mikezappa87](https://github.com/mikezappa87)**) became new
+      Chairs (joining Tim Hockin (**[@thockin](https://github.com/thockin)**), who
+      remained a Chair)
+
+    - Dan Winship (**[@danwinship](https://github.com/danwinship)**) and
+      Antonio Ojea (**[@aojea](https://github.com/aojea)**) became SIG Network's
+      first Tech Leads.
+
+  - Major KEPs? FIXME
+
+  - Gateway API
+
+    [Gateway API is now GA!](https://kubernetes.io/blog/2023/10/31/gateway-api-ga/)
```

Comment:
v1.1 blog post should be coming out today as well

Reply:
That's for _next year's_ annual report :slightly_smiling_face: